File: SRTI_Source/Java_Server_Client_Source/CustomDataTransfer_20171213/src/mainServer/RTISim.java
package mainServer;

/*
 * RTISim.java
 * 
 * - if you want a connected sim to receive callback function calls to immediately handle received messages,
 * 		the simulation must extend from this class and implement these two functions.
 * */

public interface RTISim {
	
	public void receivedMessage(String messageName, String content, String timestamp, String source);
	
	public String getSimName();
}

File: SRTI_Source/C++_Client_Source/RTILib_C++_20180313/RTISim.cpp
/*
	RTISim.cpp
	- if you want a connected sim to receive callback function calls to immediately handle received messages,
		the simulation must extend from this class and implement these two functions.

*/



#include "stdafx.h"
#include "RTISim.h"
#include <string>

using namespace std;

#ifndef CPPADDED
#define CPPADDED
#endif

RTISim::RTISim()
{
}

RTISim::~RTISim()
{
}
